src/components/profile/ProfileSetPassword.jsx

```diff
@@ -84,9 +88,11 @@ export default function ProfileSetPassword() {
                     type="text"
                     placeholder="new password"
```

Comment:
see https://stackoverflow.com/a/71429960 for example

Reply:
If change to use validate, it will break the logic of lively show if the password matches. I changed to use watch in the new version.